README.md

```diff
@@ -20,4 +20,19 @@ A fragment may be reasonable to insert into multiple locations. However, downstr
 
 ## Files produced
 
-The plugin will generate two files, one which is a `Phylogeny[Rooted]` type, and once which is a `Placements` type. The former is the tree with the sequences placed (which could be inserted), and are identified by their corresponding sequence IDs. The latter is a JSON object which, for every input sequence, describes the different possible placements. 
+The plugin will generate two files, one which is a `Phylogeny[Rooted]` type, and onc which is a `Placements` type. The former is the tree with the sequences placed (which could be inserted), and are identified by their corresponding sequence IDs. The latter is a JSON object which, for every input sequence, describes the different possible placements.
+
+### Workaround for Q2 plugin error
+
+You might encounter the following error when using the resulting `Phylogeny[Rooted]` tree artifact for ``qiime diversity beta-phylogenetic`` computations:
```

Comment:
This issue is a sensitivity of `scikit-bio`'s validation prior to executing UniFrac. One work around would be to either address it in the reference tree or to normalize before writing it out. It would be great to not have to ask users to resolve this as it will alienate quite a few people quickly.

Reply:
I think the fastest solution is to do some post-processing of the newick tree before the plugin returns.
```
tree_ref = TreeNode.read('tree.nwk')
for node in tree_ref.preorder():
    if node.length is None:
        node.length = 0
tree_ref.write('tree.nwk')
```
Might be easier than asking Siavash to update the reference tree in his package and create a new release.

Reply:
That works, and we can make it better later.